packages/helm-charts/prometheus-stackdriver/values.yaml

```diff
@@ -138,6 +138,31 @@ remote_write:
               |storage_.+\
               |workqueue_.+)"
       source_labels: [__name__]
+    - action: drop
```

Comment:
Since this has grown into quite the long list (combined with the above), is there a way to present it by grouping the dropped metrics? A starting point can be alphabetic, logical groups would be even better.

Reply:
Ordered alphabetically